deploy/role.yaml

```diff
@@ -17,6 +16,7 @@ rules:
   - configmaps
   - secrets
   - serviceaccounts
+  - namespaces
```

Comment:
It might be better to have `namespaces` in a separate group, with permissions to only `list` and read`.